### PR TITLE
Update the Orthogonal slice module to use a manually created UI

### PR DESCRIPTION
**Pull request by mathturtle**
This allows greater control of UI elements.  This change gives the
slider to set the current slice index more space and sets its page size
(delta when clicked on above or below the slider) to 1.

For #449.  @Hovden try this out.

### CODE REVIEW

**Comment by Hovden** [comment]
If the code is good, we can merge.

**Comment by yijiang1** [comment]
@Hovden new UI on Mac look like this
screen shot 2016-08-24 at 11 24 29 am

**Comment by Hovden** [comment]
Looks good.

File: tomviz/IntSliderWidget.cxx [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+  {
+    QObject::connect(this->LineEdit, SIGNAL(textChanged(const QString&)),
+                     this, SLOT(textChanged(const QString&)));
+    QObject::connect(this->LineEdit, SIGNAL(textChangedAndEditingFinished()),
+                     this, SLOT(editingFinished()));
+  }
+  
+}
+
+//-----------------------------------------------------------------------------
```
Comment (cquammen) [inline tomviz/IntSliderWidget.cxx]: We're not putting in these horizontal breaks anymore.

File: tomviz/IntSliderWidget.cxx [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+  this->StrictRange = false;
+
+  QHBoxLayout* l = new QHBoxLayout(this);
+  l->setMargin(0);
+  this->Slider = new QSlider(Qt::Horizontal, this);
+  this->Slider->setRange(this->Minimum, this->Maximum);
+  l->addWidget(this->Slider, 4);
+  this->Slider->setObjectName("Slider");
+  if (showLineEdit)
+  {
```
Comment (cquammen) [inline tomviz/IntSliderWidget.cxx]: Opening brace is preferred to be at the end of the line above. Throughout.

File: tomviz/IntSliderWidget.cxx [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+    this->LineEdit = new pqLineEdit(this);
+    l->addWidget(this->LineEdit);
+    this->LineEdit->setObjectName("LineEdit");
+    this->LineEdit->setValidator(new QIntValidator(this->LineEdit));
+    this->LineEdit->setTextAndResetCursor(QString().setNum(this->Value));
+  }
+  else
```
Comment (cquammen) [inline tomviz/IntSliderWidget.cxx]: `} else {` is preferred, I believe

File: tomviz/IntSliderWidget.cxx [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+//-----------------------------------------------------------------------------
+int IntSliderWidget::value() const
+{
+  return this->Value;
+}
+
+//-----------------------------------------------------------------------------
+void IntSliderWidget::setValue(int val)
+{
+  if(this->Value == val)
```
Comment (cquammen) [inline tomviz/IntSliderWidget.cxx]: `if(` -> `if (` throughout

**Comment by cquammen** [comment]
Works great. Scroll wheel works nicely, too. Only minor code style changes are requested.

**Comment by cryos** [comment]
Let's merge and then we can simply use clang-format on all the source to get it to a consistent state. Please remove the separators as they will be missed.

**Comment by mathturtle** [comment]
Ok, I've removed the separators.

**Comment by cryos** [comment]
Thanks, merging and planning on running clang-format on all source files soon.

**Pull request merged**